caravel/assets/javascripts/SqlLab/components/TabbedSqlEditors.jsx

```diff
@@ -122,9 +122,10 @@ class TabbedSqlEditors extends React.PureComponent {
             <MenuItem eventKey="2" onClick={this.renameTab.bind(this, qe)}>
               <i className="fa fa-i-cursor" /> rename tab
             </MenuItem>
-            <MenuItem eventKey="3">
+            {qe && <MenuItem eventKey="3">
```

Comment:
i think this is more readable when the condition and component are on separate lines:

```jsx
{qe && 
  <MenuItem eventKey="3">
    <i className="fa fa-clipboard" /> <CopyQueryTabUrl queryEditor={qe} />
  </MenuItem>
}
```